general_docs/glossary.adoc

```diff
@@ -1,12 +1,89 @@
+[glossary]
 = Terms and definitions
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-Et malesuada fames ac turpis egestas.
-Mauris pharetra et ultrices neque ornare aenean euismod elementum nisi.
-Nulla facilisi morbi tempus iaculis urna id.
-Justo eget magna fermentum iaculis.
-Sed augue lacus viverra vitae congue eu.
-Velit ut tortor pretium viverra suspendisse.
-Amet commodo nulla facilisi nullam vehicula ipsum a.
-Nibh nisl condimentum id venenatis a.
-Diam vel quam elementum pulvinar etiam non quam lacus suspendisse.
\ No newline at end of file
+//Writing style adopted from https://www.iso.org/glossary.html
+CS (co-simulation)::
+Type of Functional Mockup Unit (FMU). +
+Co-simulation FMUs contain their own numerical solver.
+This importing tool sets the FMU inputs, asks the FMU to step forward a given time, and reads the FMU output after the step is completed.
+With OSMP, models are packaged as valid _FMI-2.0 FMU for co-simulation_ +
++
+See also: https://www.modelon.com/functional-mock-up-interface-fmi/
+
+Ego vehicle::
+Externally controlled vehicle used for scenario descriptions. +
+For evaluation of automated driving systems, the ego vehicle is the vehicle that is controlled by the system-under-test.
+For human-driver experiments, the ego vehicle is the vehicle that is driven by the human driver.
+
+Environmental effect model::
+//TODO: input needed
```

Comment:
OSI CCB 15.09.
@pasched Look at the documentation from OSMP regarding the model types

Reply:
@stefancyliax  I had a look at the definitions of the model types in OSMP (chapter 4.2.1) and the terms and definition chapter. The definitions do not contradict each other. OSMP definitions are a bit shorter and more technical regarding the package information.
The only thing I noticed is that there is an introductory sentence needed for "Logical model" in the "terms and definitions". I would suggest to take the sentence from chapter 3.2.3 "Logical models consume sensor-data messages and produce sensor-data messages."

Reply:
Ok then please do.